### DISCUSSION TITLE
New Controlling plugin: migrate from old MR Software plugin

### DISCUSSION THREAD

**Discussion by idemus** [kimai/Paid-plugins#110] [Questions & Answers] [answered]
Hi Kevin,
I've read in your changelog that there is a possibility since 18.9. to migrate the old data from MR plugin to your new plugin. But I can't find any documentation for this. Could you give me a hint, please.

Best regards
Klaus

**Comment by pbksol** [comment, marked as answer]
See https://www.kimai.org/documentation/plugin-contract.html

**Comment by idemus** [comment]
Hi Pascal,
thank you very much. It worked perfectly! Don't know why I haven't found it.

Best regards
Klaus
